src/cmds/scripts/pbs_postinstall.in

```diff
@@ -806,7 +821,7 @@ create_home() {
 echo "*** PBS Installation Summary"
 echo "***"
 echo "*** Postinstall script called as follows:"
-echo "*** $0 $*"
+printf "*** $0 "; printf "%q " "$@"; printf "\n"
```

Comment:
I see. PP-441 had a bug in it, and now you are fixing that bug and checking the bug fix in as part of this feature. Can you please file a PP- ticket (i.e. a bug in Open Jira) for this bug that you are fixing? It will be helpful for tracking purposes.